package.json

```diff
@@ -55,6 +55,7 @@
     "@google-cloud/promisify": "^2.0.0",
     "@types/stack-trace": "0.0.29",
```

Comment:
Adding `"@types/big.js": "^4.0.5",` here should fix the problem where the system tests are complaining about missing types for `big.js`.

Reply:
Thanks heaps @olavloite. That did indeed fix the issue.